vscode/extension/src/lsp/custom.ts

```diff
@@ -41,7 +42,19 @@ interface AllModelsRequest {
 
 interface AllModelsResponse extends BaseResponse {
   models: string[]
+  model_completions: ModelCompletion[]
   keywords: string[]
+  macros: MacroCompletion[]
+}
+
+interface ModelCompletion {
+  name: string
+  description?: string
+}
+
+interface MacroCompletion {
+  name: string
+  description?: string
 }
```

Comment:
The reason for this not being updated, is that it is legacy and so it really should just be dropped.